README.rst

```diff
@@ -47,11 +47,13 @@ This plugin adds the following options to the csv-table_ directive:
     This option takes a Python dict specifying the column index (starting at
     zero) and a regular expression. Rows are included if the columnar value
```

Comment:
Let's rephrase the sentence as is:

`Rows are included if the columnar value matches *all* supplied regular expression.`